### ISSUE TITLE
Re11-worldwise > src > components > City | unable to get id

### ISSUE THREAD

**Issue by devYuraKim** [devYuraKim/react#20]
It seems getting `id` isn't working

<img width="731" alt="Screenshot 2024-09-12 at 1 36 48 PM" src="https://github.com/user-attachments/assets/2b279c76-5128-4a43-8cec-f677010a6a3c">

```
const formatDate = (date) =>
  new Intl.DateTimeFormat("en", {
    day: "numeric",
    month: "short",
    year: "numeric",
  }).format(new Date(date));

function City() {
  const { id } = useParams();
  const { currentCity, getCurrentCity } = useCities();

  useEffect(
    function () {
      getCurrentCity(id);
      console.log(currentCity);
    },
    [id]
  );

  const { emoji, cityName, date, notes } = currentCity;

  return (
    <div className={styles.city}>
      <div className={styles.row}>
        <h6>City name</h6>
        <h3>
          <span>{emoji}</span> {cityName}
        </h3>
      </div>

      <div className={styles.row}>
        <h6>You went to {cityName} on</h6>
        <p>{formatDate(date || null)}</p>
      </div>

      {notes && (
        <div className={styles.row}>
          <h6>Your notes</h6>
          <p>{notes}</p>
        </div>
      )}

      <div className={styles.row}>
        <h6>Learn more</h6>
        <a
          href={`https://en.wikipedia.org/wiki/${cityName}`}
          target="_blank"
          rel="noreferrer"
        >
          Check out {cityName} on Wikipedia &rarr;
        </a>
      </div>

      <div>{/* <ButtonBack /> */}</div>
    </div>
  );
}

```

**Comment by devYuraKim** [comment]
TL;DR: check parameter name set in routes
---
The parameter name was set as '`cityId`' in the routes, but I called it '`id`' in the `City` component.

`App` component
```
<Route path="cities/:cityId" element={<City />} />
```

`City` component
```
 const { id } = useParams();
  const { currentCity, getCurrentCity } = useCities();

  useEffect(
    function () {
      getCurrentCity(id);
      console.log(currentCity);
    },
    [id]
  );
```
